content/authors/Wenxuan-Cheng/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cheng Wenxuan
+user_groups:
+- Undergraduate Students
+
+authors:
+- Wenxuan Cheng
+bio: 
+
+education:
+  courses:
+  - course: B.Soc.Sci. (Hons.), Psychology
+    institution: Nanyang Technological University, Singapore
+    year: 2017 - 2021
+
+email: "CHEN1176@e.ntu.edu.sg"
+interests:
+- Cognitive neuroscience
+- Clinical neuropsychology
+
+
+organizations:
+- name: Nanyang Technological University
+  url: ""
+role: FYP student
```

Comment:
hi @chen1176 , 

Please change the role to "Undergraduate Student" to standardize with the rest

Reply:
Hi I have updated the file, thank you!